docker-compose-live.yml

```diff
@@ -1,31 +0,0 @@
-version: "3.6"
```

Comment:
I am confident that this docker-compose-live.yml file is unused and it appears to not work. If something breaks because of this I'll just restore it but I feel strongly that this file is unused.